app/views/applications/index.html.erb

```diff
@@ -3,7 +3,7 @@
 
   <% if @application.present?  %>
     <p><%= current_user.last_name %>, <%= current_user.first_name %> - <%= link_to 'View application', "/applications/#{@application.id}"%></p>
+  <% elsif %>
+    <p><%= link_to 'Start New Application', 'applications/new' %></p>
```

Comment:
Good habit to get into using the generated method that Rails gives you for the various paths. Use `new_application_path` in place of `'applications/new'`. So in the end, you have:

```ruby
<%= link_to 'Start New Application', new_application_path %>
```